database/db.go

```diff
@@ -229,11 +242,18 @@ func NewDbFromConfig(c *Config, logger *logging.Logger, connectorCallbacks Retry
 	}, nil
 }
 
-// GetAddr returns the database host:port or Unix socket address.
+// GetAddr returns a URI-like database connection string of the following syntax: type[+tls]://user@host[:port]/database
```

Comment:
We could render the syntax as code:

```suggestion
// GetAddr returns a URI-like database connection string of the following syntax:
//
//	type[+tls]://user@host[:port]/database
```

<img width="635" alt="Screenshot 2024-10-07 at 12 58 10" src="https://github.com/user-attachments/assets/07672678-81ed-4cbe-a8dd-be12076ed674">

But I don't care too much 😉. (Same for Redis.)